### PR TITLE
[1LP][RFR] Automate GIT Domain import/export

### PR DESCRIPTION
**Pull request by mfalesni**
QuantifiedCode issues addressed where it was possible, the rest are by design.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a8a1344da35c25dd6fc97683b73085899f53ca30:

`cfme/automate/explorer/domain.py`:
- :red_circle: Line 2:1: **F401** *'re' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2ba14e29cd5288a39c67085548ed56f228175f5f:

`cfme/automate/explorer/domain.py`:
- :red_circle: Line 2:1: **F401** *'re' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/base/ui.py [hunk 1]
```diff
@@ -1084,3 +1084,54 @@ def step(self):
...
             *automate_menu_name(self.obj.appliance) + ['Simulation'])
+
+
+class AutomateImportExportBaseView(BaseLoggedInPage):
+    flash = FlashMessages('div.import-flash-message')
+    title = Text('.//div[@id="main-content"]//h1')
+
+    @property
+    def in_import_export(self):
+        from cfme.automate import automate_menu_name
```
Comment (quarckster) [inline cfme/base/ui.py]: So each time when you call `in_import_export` this import will be done? Why not put it on top of the module?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/base/ui.py (hunk 1 above)
Comment (mfalesni) [inline cfme/base/ui.py]: possibly

Comment (mfalesni) [inline cfme/base/ui.py]: Actually, it is the same as in other places, I guess that is due to risk of circular imports.

**Pull request merged**